Admin/AdminSessionModule.php

```diff
@@ -327,6 +327,37 @@ protected function runLoginCallbacks()
 	}
 
 	// }}}
+	// {{{ protected function isActiveUser()
+
+	/**
+	 * Checks to see if the AdminUser is an active account.
+	 *
+	 * Users are inactive if they haven't logged in the last 90 days, or 90
+	 * days from the creation of the account if the user has never logged in.
+	 *
+	 * @param AdminUser $user_id the user to record login history for.
+	 */
+	protected function isActiveUser(AdminUser $user)
+	{
+		$active_user = false;
+
+		$threshold = new SwatDate();
+		$threshold->subtractDays(90);
+
+		if ($user->createdate instanceof SwatDate) {
+			$comparison_date = $user->createdate;
+		} elseif ($user->most_recent_login instanceof AdminUserHistory) {
+			$comparison_date = $user->most_recent_login->login_date;
+		}
```

Comment:
If createdate is null and there is no recent login date the `comparison_date` will be undefined and will cause a PHP fatal error when we call `->after()`